### ISSUE TITLE
There is a question when I run Step 3

### ISSUE THREAD

**Issue by EricerYang** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#255]
Hello, I am running step 3, how much will it run iter after it stops ?

**Comment by Daniil-Osokin** [comment]
Hi! It is going to be `420000` iterations. However, you may pick the weights earlier, when validation accuracy stops increasing.

**Comment by EricerYang** [comment]
I noticed that you provided a Loss Image in the previous issue. But the horizontal coordinate is "epoch". Did you calculate "epoch" by using "iter" alone?


> 你好！这将是`420000`迭代。但是，当验证准确度停止增加时，您可以更早地选择权重。

**Comment by EricerYang** [comment]
> Hi! It is going to be `420000` iterations. However, you may pick the weights earlier, when validation accuracy stops increasing.

I noticed that you provided a Loss Image in the previous issue. But the horizontal coordinate is "epoch". Did you calculate "epoch" by using "iter" alone?
![image](https://user-images.githubusercontent.com/57659040/162664573-e9e38a98-4833-4d64-811e-4fdc1cd69e02.png)

**Comment by EricerYang** [comment]
> I have plotted the losses, after each epoch. Each epoch corresponds to a fixed number of iterations.

Oh, thank you for your recovery. I would like to know how I can generate heat maps during training. Do you have any suggestions?

**Comment by EricerYang** [comment]
Oh, thank you for your recovery. I would like to know how I can generate heat maps during training. Do you have any suggestions?

**Comment by Daniil-Osokin** [comment]
You may check #235 discussion, it has an example.

**Comment by Daniil-Osokin** [comment]
Hope, it helped.
